doc_src/history.txt

```diff
@@ -24,13 +24,15 @@ The following commands are available:
 
 - `-v` or `--save` saves all changes in the history file. The shell automatically saves the history file; this option is provided for internal use.
 
-- `-l` or `--clear` clears the history file. A prompt is displayed before the history is erased asking you to confirm you really want to clear all history.
+- `-l` or `--clear` clears the history file. A prompt is displayed before the history is erased asking you to confirm you really want to clear all history unless you use the `builtin` prefix.
```

Comment:
Since "prefix" is a word we encounter elsewhere in the context of using the history builtin, find a different way to describe using the builtin?